Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingManager.cs

```diff
@@ -48,7 +50,35 @@ public partial class SpatialMappingManager : Singleton<SpatialMappingManager>
         /// <summary>
         /// The current source of spatial mapping data.
         /// </summary>
-        public SpatialMappingSource Source { get; private set; }
+        public SpatialMappingSource Source
+        {
+            get { return source; }
+
+            private set
+            {
+                if (source != value)
+                {
+                    UpdateRendering(false);
+
+                    var oldSource = source;
```

Comment:
Mostly I like to know what the file type is so it's easier to understand what's going on.  If it's not important, it's no big deal.